### DISCUSSION TITLE
Adding a comment with a git push

### DISCUSSION THREAD

**Discussion by afargnoli-ops** [godotengine/godot-proposals#5523] [General Discussions]
What I've found frustrating is that after I've made several commits on my local machine, implemented, tested, and reviewed some enhancement, I can't include a comment with my push to say something along the lines of "These commits add the ABC feature to our wonderful product." It just seems that a comment with a push would be something many people would use.

**Comment by miv391** [comment]
That's not how git works. Commit has a comment (or message). Push doesn't have a comment, push just pushes the commits to another repo.

**Reply by afargnoli-ops** [reply]
I'm suggesting a change that I believe would be enormously useful.

**Comment by Zireael07** [comment]
Not godot related at all btw

**Reply by afargnoli-ops** [reply]
Is there a different forum for proposing enhancements to Git? This was the only one I found.

**Reply by Calinou** [reply]
Git has no forum for feature suggestions. Discussions related to Git development are made on its mailing list, but the barrier to entry is very high for new features in Git nowadays. Since it's a mature version control system, most of its developers aren't so willing to dabble with experimental ideas.
